packages/web/pages/user/my-account/orders.js

```diff
@@ -0,0 +1,350 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'react-i18next';
+import { FiMessageSquare, FiEye, FiX, FiCheck } from 'react-icons/fi';
+import { UserProfile } from '../../../components/UserProfile';
+import { Protected } from '../../../components/Authorization';
+import { Title } from '../../../components/Common';
+import { DataGrid } from '../../../components/DataGrid';
+import { IconButton } from '../../../components/Button';
+import { ORDERING as orderEnum } from '../../../utils/enums/api.enum';
+import { dateToString } from '../../../utils/helper';
+import { STATUS as dealStatusEnum } from '../../../utils/enums/orders.enum';
+import { useModal } from '../../../hooks';
+import OrderMessages from '../../../components/OrderMessages';
+
+const sortOptions = [
+	{ value: 'title', label: 'Título' },
+	{ value: 'responsible', label: 'Responsável' },
+	{ value: 'status', label: 'Status' },
+	{ value: 'order_date', label: 'Data do pedido' },
+];
+const itemsPerPage = 5;
+
+const ordersMock = [
+	{
+		id: 1,
+		title: 'Tecnologia Um',
+		quantity: 1,
+		buyer: 'Fulano',
+		status: 'deal_struck',
+		created_at: '2020-11-09 12:53:24.000000',
+	},
+	{
+		id: 2,
+		title: 'Tecnologia Dois',
+		quantity: 123,
+		buyer: 'Beltrano',
+		status: 'deal_ongoing',
+		created_at: '2020-11-19 12:53:24.000000',
+	},
+	{
+		id: 3,
+		title: 'Tecnologia Tres',
+		quantity: 413,
+		buyer: 'Ciclano',
+		status: 'deal_cancelled',
+		created_at: '2020-03-14 12:53:24.000000',
+	},
+];
```

Comment:
Why do you have a mock here? Is the API not ready yet? :eyes:

Reply:
Yeah, it's still in progress :)